src/containers/ViewerPanel/index.tsx

```diff
@@ -392,22 +392,24 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
                         this.onTrajectoryFileInfoChanged
                     }
                 />
-                <PlaybackControls
-                    playHandler={this.startPlay}
-                    time={time}
-                    timeStep={timeStep}
-                    displayTimes={displayTimes}
-                    timeUnits={timeUnits}
-                    onTimeChange={this.skipToTime}
-                    pauseHandler={this.pause}
-                    prevHandler={this.playBackOne}
-                    nextHandler={this.playForwardOne}
-                    isPlaying={isPlaying}
-                    firstFrameTime={firstFrameTime}
-                    lastFrameTime={lastFrameTime}
-                    loading={isBuffering}
-                    isEmpty={status === VIEWER_EMPTY}
-                />
+                {firstFrameTime !== lastFrameTime && (
```

Comment:
i wonder if there is a better check, like count of number of frames === 1

Reply:
yeah, this component doesn't know the total number, and I didn't investigate when/if that gets sent in. I don't see any way this would be wrong when the count would be right though